workers/tests/unittests/test_raster.py

```diff
@@ -20,23 +22,20 @@ def setUp(self):
         with open(path, "r") as f:
             self.feature = geojson.load(f)
 
-    @mock.patch(
-        "ohsome_quality_analyst.raster.client.get_data_dir",
-        return_value="test_dir",
-    )
-    @mock.patch("os.path.exists", return_value=True)
+    @mock.patch("ohsome_quality_analyst.raster.client.DATA_DIR", tempfile.gettempdir())
     def test_get_raster_path(self, *args):
+        Path(
+            tempfile.gettempdir() + "/GHS_BUILT_LDS2014_GLOBE_R2018A_54009_1K_V2_0.tif"
+        ).touch()
+
         self.assertEqual(
             raster_client.get_raster_path(self.raster_dataset),
             os.path.join(
-                "test_dir", "GHS_BUILT_LDS2014_GLOBE_R2018A_54009_1K_V2_0.tif"
+                tempfile.gettempdir(),
+                "GHS_BUILT_LDS2014_GLOBE_R2018A_54009_1K_V2_0.tif",
             ),
         )
```

Comment:
Tests shouldn't mess with files in my `/tmp`. Please always create a tempdir/-file and remove it after the tests are done.

Reply:
Now a tmp dir get created for this test